app/v1/models/response/retrievePropertyBsas/AccountingPeriod.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package v1.models.response.retrieveBsas
+package v1.models.response.retrievePropertyBsas
```

Comment:
Can this package renaming be undone? It's completely incompatible with how I've structured the models in the self-employment endpoint, since I'm reusing some of the models from this endpoint.

Reply:
I've handled the splitting up of ukProperty vs selfEmployment in my PR and it'd be easier for you to rename the package again than it will for me to rebase and sort out a merge mess